src/components/react-hook-form/utils/cancel-button.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Button } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+const CancelButton = ({
+    onClick,
+    variant,
+    disabled,
+    withCustomColor = false,
+}) => {
+    return (
+        <Button
+            onClick={onClick}
+            variant={variant}
+            disabled={disabled}
+            color={withCustomColor ? 'primary' : 'secondary'}
```

Comment:
Don't use secondary it already exist

Reply:
In the app, "secondary" refers to the pink color. I've overloaded this parameter because the name "secondary" is more descriptive than anything else, but I can change it if you have another name in mind

Reply:
I replaced it with **customButton**